papas/aliceproto/PFfloodfill.py

```diff
@@ -0,0 +1,82 @@
+
+
+class Node(object):
+    '''Basic interface for nodes traversed by the floodfill algo.
+    Floodill will give a label to each node corresponding to the 
+    disconnected subgraph it corresponds to. 
+
+    The linked attribute is a list that should contain the elements y-linkes.
+    '''
+    def __init__(self):
+        self.linked = []
+        self.block_label = None
+
+    def accept(self, visitor):
+        '''Called by visitors, such as FloodFill.'''
+        notseen = visitor.visit(self)
+        if notseen:
+            for elem in self.linked:
+                elem.accept(visitor)
+                
+class Block(object):
+    
+    def __init__(self, label=None):
+        self.linked = []
+        self.block_label = label
+        self.distances = []
+
+    def append(self, element):
+        self.linked.append(element)
+        
+    def dosort(self):
+        self.linked.sort(key=self.getType)
+        
+        
+    def getType(self,thing):
+        return thing.type.value
+                
+
+
+class FloodFill(object):
```

Comment:
why a floodfill here and a DAGfloodfill above? 
and there is also mine in the pfalgo package. 
need to see if we can work with a single floodfill algo